apps/linkedin-to-notion/src/components/LinkedInNotionSidePanel/NotionDatabasesSelect.tsx

```diff
@@ -7,7 +7,11 @@ import { useStorage } from '@plasmohq/storage/hook';
 
 import { getDatabaseTitle } from './utils/notionFormat.util';
 
-export const NotionDatabasesSelect = () => {
+type NotionDatabasesSelectProps = {
+  valueChangeHandler: (value: string) => void;
+};
+
+export const NotionDatabasesSelect = ({ valueChangeHandler }: NotionDatabasesSelectProps) => {
```

Comment:
I'd rather call it `onValueChange` but I think we alreadyhad this debate so fine for me if you want to keep it like this 😉